packages/indexer/src/otel.ts

```diff
@@ -1,3 +1,30 @@
-import { trace } from "@opentelemetry/api";
+import { metrics, trace } from "@opentelemetry/api";
 
-export const tracer = trace.getTracer("@apibara/indexer");
+export function createTracer() {
+  return trace.getTracer("@apibara/indexer");
+}
+
+export function createIndexerMetrics() {
+  const meter = metrics.getMeter("@apibara/indexer");
+
+  const currentBlockGauge = meter.createGauge("current_block", {
```

Comment:
IIRC the opentelemetry collector converts the metrics name. After you get the end to end example running can you message me the metric name on Slack?